src/routes/about/routes.rs

```diff
@@ -101,5 +101,9 @@ pub async fn backbone(
 #[doc(hidden)]
 #[get("/about/redis")]
 pub async fn redis(context: RequestContext, _token: AuthorizationToken) -> ApiResult<String> {
-    Ok(context.cache().info().await.unwrap_or(String::new()))
+    Ok(context
+        .cache(UNSPECIFIED_CHAIN)
```

Comment:
Potentially we should add the chain to the url here

Reply:
For an hotfix I would not do it. The amount of changes is already quite broad.